integration/lerna-publish-canary.test.js

```diff
@@ -22,7 +22,7 @@ const env = {
 
 test("lerna publish --canary uses default prerelease id", async () => {
   const { cwd } = await cloneFixture("normal");
-  const args = ["publish", "--canary", "--yes"];
+  const args = ["publish", "--canary", "--yes", "--no-temp-tag"];
```

Comment:
I had to add `--no-temp-tag` to all the end-to-end tests because we don't have a good solution for local testing with an _actual_ registry, and none of the test packages are actually "live" on the public registry.